_layouts/person.html

```diff
@@ -156,7 +156,9 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
       </div>
 
       <h3 class="person-show-title">
+       <a href="{{ record.item.url }}">
```

Comment:
Can we combine this link with the poster one (i.e. bump it up a level)? Neater and better for tab order and other things.